### DISCUSSION TITLE
Manual update through copy&paste of extracted installer files

### DISCUSSION THREAD

**Discussion by citizen-gh** [valinet/ExplorerPatcher#3704] [Q&A]
Is there a way to update EP without running the installer, for example by copy&pasting the extracted files? What is to be considered?
My challenge is that there is enterprise security software running which recognizes EP as a malware since some months and I can't run the installer anymore. I've installed it last year and run an older version. However, I can copy&replace existing files. Thanks in advance! And thanks to Valentin for this great tool!

**Comment by Amrsatrio** [comment]
Yes, of course, it's possible, but we don't have a document outlining that for now. You have to look at the ep_setup.c source code and find lines with InstallResource( to see which files it drops at where.

**Reply by citizen-gh** [reply]
Thank you! Is it safe to consider that there are no registry changes needed when replacing files with newer versions or other kinds of deep under the hood adjustments? I've already crashed my windows, respectively a windows updates was incompatible with my not up-to-date version last year, so it needed a reinstall, I don't want to risk again.

**Reply by Amrsatrio** [reply]
No. The new code should handle the old registry accordingly.
